src/main/kotlin/org/rust/lang/core/types/ty/TyTuple.kt

```diff
@@ -18,5 +18,7 @@ data class TyTuple(val types: List<Ty>) : Ty(mergeFlags(types)) {
         types.any(visitor)
 
     override fun toString(): String = tyToString(this)
+
+    override val isCopyable = types.fold(true) { acc, ty -> acc && ty.isCopyable }
```

Comment:
Would this also work as `types.all { it.isCopyable }` ?
(Same at other places)

Reply:
yep, `types.all` is better.